savoirfairelinux/sous-chef
# Deleting a route do not delete its clients

## Fixes #585 

### Changes proposed in this pull request:

* Use on_delete attribute inside the models to change the default behavior when we delete a foreign key
* When the route is deleted, the attribute route of the client is set to NULL
* Add a test for this case

### Status


### How to verify this change

* Go to the Django backend
* Select the Routes model
* Delete a route (ex: McGill West)
* Verify for a client who had McGill West for route if it still exist and if it's route is null.

### Deployment notes and migration


### New translatable strings
